client/src/components/overrides.d.ts

```diff
@@ -0,0 +1,31 @@
+import { Overrides } from '@material-ui/core/styles/overrides';
+import { MuiPickersOverrides } from '@material-ui/pickers/typings/overrides';
+
+type overridesNameToClassKey = {
+    [P in keyof MuiPickersOverrides]: keyof MuiPickersOverrides[P];
+};
+
+declare module '@material-ui/core/styles/overrides' {
+    export interface ComponentNameToClassKey extends overridesNameToClassKey { }
+}
+// import { MuiPickersComponentsToClassName } from '@material-ui/pickers/src/typings/overrides';
```

Comment:
delete these if you don't need them

Reply:
Funny story. Never touched this file. It's weird that it's actually here. I'll get rid of it